pkg/cmd/init/options.go

```diff
@@ -18,12 +18,13 @@ import (
 const jsonOutput = "json"
 
 type InitOptions struct {
-	TemplateNameOrURL string
-	Online            bool
-	ProjectName       string
-	Force             bool
-	Yes               bool
-	CustomParamsJSON  string
+	TemplateRepoPath string
```

Comment:
`TemplateNameOrURL` is miss understanding, that's my fault, this field contains online/offline mode, online is url, offline is local dir.
```suggestion
	TemplatePathOrURL string
````

Reply:
that's right, but TemplatePathOrURL is not elegant either. TemplateRepoPath is used to refer to the path which can get template repo, can be url or local path.

Reply:
`TemplateRepoPath` is a path, when does `path` means URL?

Reply:
haha, the original code is also provided by me. I changed it because maybe there will be validation when online is true. Your suggestion is better for now, I'll change it.

Reply:
That’s what I want to express, is it confusing?